_examples/golang-sse/proto/chat.gen.go

```diff
@@ -603,6 +617,11 @@ func RequestFromContext(ctx context.Context) *http.Request {
 	r, _ := ctx.Value(HTTPRequestCtxKey).(*http.Request)
 	return r
 }
+
+func AnnotationsFromContext(ctx context.Context) map[string]string {
```

Comment:
call this MethodAnnotations?

Reply:
Random idea: Since we're adding more and more context keys / helpers similar to this one, maybe it would be useful to have just one ctx key / helper, which would return struct with all the values?

```go
method := proto.MethodFromContext(ctx)
// method.ServiceName
// method.Name
// method.Annotations
```

Reply:
Like this: https://github.com/webrpc/gen-golang/pull/71/commits/20a35c534b370b41b4c69dd8c37270174d28b3c1

??

Reply:
```
func MethodFromContext(ctx context.Context) MethodCtx {
	name, _ := ctx.Value(MethodNameCtxKey).(string)
	service, _ := ctx.Value(ServiceNameCtxKey).(string)
	annotations, _ := ctx.Value(methodAnnotationsCtxKey).(map[string]string)

	return MethodCtx{
		Name:        name,
		Service:     service,
		Annotations: annotations,
	}
}
```